src/neptune/attributes/series/fetchable_series.py

```diff
@@ -35,20 +35,24 @@
 
 Row = TypeVar("Row", StringSeriesValues, FloatSeriesValues)
 
+MAX_FETCH_LIMIT = 1000
+
 
 class FetchableSeries(Generic[Row]):
     @abc.abstractmethod
     def _fetch_values_from_backend(self, offset, limit) -> Row:
         pass
 
-    def fetch_values(self, *, include_timestamp: bool = True, progress_bar: Optional[ProgressBarType] = None):
+    def fetch_values(
+        self,
+        *,
+        include_timestamp: bool = True,
+        progress_bar: Optional[ProgressBarType] = None,
```

Comment:
Let's maybe move the `progress_bar` to the bottom of the function signature, as it will likely be the least-often set argument